stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/connection.bal

```diff
@@ -9,3 +21,37 @@ public function <Connection conn> respondContinue () (HttpConnectorError) {
     HttpConnectorError err = conn.respond(res);
     return err;
 }
+
+@Description { value:"Sends a redirect response code to the user with given redirection status code." }
```

Comment:
"code" in "redirect response code" sounds outplace here.. Shall we just say redirect response?